package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "date-fns": "^2.30.0",
```

Comment:
Both `date-fns` and `moment` are not used at all. Please remove them from the package.json and then run `npm install` to update the `package-lock.json` automatically.